lm_eval/base.py

```diff
@@ -713,9 +713,12 @@ def fewshot_context(
                 fewshotex = [x for x in fewshotex if x != doc][:num_fewshot]
 
             labeled_examples = (
-                "\n\n".join(
+                "\n\n" +
+                "".join(
                     [
-                        self.doc_to_text(doc) + self.doc_to_target(doc)
+                        self.doc_to_text(doc) + "\n[/INST]\n" + self.doc_to_target(doc) + "\n</s>\n\n<s>\n[INST]\n"
+                        if os.environ.get("USE_LLAMA_TEMPLATE", "") == "yes"
```

Comment:
Here and below use `ON`/`OFF` instead of "yes" for `USE_UPDATED_SCORER`.
And `LLAMA` for `USE_TEMPLATE` (instead of `USE_LLAMA_TEMPLATE`), it gives us more flexibility with other model templates